src/rpc/format/cbor/convert.rs

```diff
@@ -54,23 +56,43 @@ impl TryFrom<Cbor> for Value {
 			Data::Tag(t, v) => {
 				match t {
 					// A literal datetime
-					TAG_DATETIME => match *v {
+					TAG_STRING_DATETIME => match *v {
 						Data::Text(v) => match Datetime::try_from(v) {
 							Ok(v) => Ok(v.into()),
 							_ => Err("Expected a valid Datetime value"),
 						},
 						_ => Err("Expected a CBOR text data type"),
 					},
+					// An epoch-based datetime (https://www.rfc-editor.org/rfc/rfc8949.html#name-epoch-based-date-time)
+					TAG_DATETIME => match *v {
+						Data::Integer(v) => match Datetime::try_from(i128::from(v)) {
+							Ok(v) => Ok(v.into()),
+							_ => Err("Expected a valid Datetime value"),
+						},
+						Data::Float(v) => match Datetime::try_from(v) {
+							Ok(v) => Ok(v.into()),
+							_ => Err("Expected a valid Datetime value"),
+						},
+						_ => Err("Expected a CBOR integer or float data type"),
+					},
```

Comment:
I'd prefer to support datetimes as `(i64, u32)`. I see issues arising from supporting `f64` values for storing datetimes, especially those with nanosecond precision. Perhaps we make this a custom tag and support a 2 element array? I'm not set on my thoughts on this though, so happy to discuss...